lib/scope.js

```diff
@@ -315,34 +317,37 @@ module.exports = {
     if ( !scope.page ) { scope.page = await scope.browser.newPage(); }
 
     // Try to go to the given page
-    let load_result = null;
+    let end_status = `page loaded`;
     // Assumption: Loading first page shouldn't take too long. Thus try again after short
     // timeout. If this is wrong, we have to rethink how we handle retrying a server that
     // has not yet finished loading.
     try {
       attempt_num = attempt_num || 0;
       attempt_num++;
       
-      await scope.page.goto(interview_url, {waitUntil: 'domcontentloaded'});
+      await scope.page.goto(interview_url, { waitUntil: 'domcontentloaded' });
       // This shouldn't be needed, but I think it may help with the ajax
       // requests. Might not solve all race conditions.
-      load_result = await scope.getLoadData( scope, { timeout: attempt_timeout });
+      let load_result = await scope.getLoadData( scope, { timeout: attempt_timeout });
+
+      if ( load_result.error ) {
+        await scope.addToReport(scope, { type: `error`, value: load_result.error });
+        expect( load_result.error ).to.equal( '' );
+      }
 
     } catch ( err ) {
       // If it doesn't work, maybe try again
       if ( attempt_num <= max_attempts ) { await scope.load( scope, file_name, attempt_num ); }
       else {
+        end_status = `page did not load`;
         // Otherwise throw an error, though there must be a better check we can do
         let msg = `The interview at ${ interview_url } did not load after ${ max_attempts } tries. The test gave the page ${ attempt_timeout/1000 } seconds to load each time. Check the screenshot artifact.`;
         await scope.addToReport(scope, { type: `error`, value: msg });
-        expect( false, msg ).to.be.true;
+        expect( end_status ).to.equal( `page loaded` );  // Can an error really be thrown in a `catch`?
       }
     }
 
-    if ( load_result.error ) {
-      await scope.addToReport(scope, { type: `error`, value: load_result.error });
-      expect( load_result.error ).to.equal( '' );
-    }
+    expect( end_status ).to.equal( `page loaded` );
```

Comment:
Instead of repeating this statement twice (once inside catch block and once outside), this might be a good place to add in a `finally` block.